src/applications/personalization/profile-2/tests/components/connected-apps/ConnectedApp.unit.spec.jsx

```diff
@@ -0,0 +1,32 @@
+import React from 'react';
+import { expect } from 'chai';
+import { shallow } from 'enzyme';
+
+import { ConnectedApp } from '../../../components/connected-apps/ConnectedApp';
+
+describe('<ConnectedApp>', () => {
+  it('renders correctly', () => {
+    const defaultProps = {
+      attributes: {
+        created: '2020-06-09T00:42:41.798Z',
+        grants: [{ title: 'hello' }, { title: 'hello' }, { title: 'hello' }],
+        logo: 'logoURL',
+        title: 'Random title',
+      },
+      confirmDelete: () => {},
+      id: '1',
+    };
+
+    const wrapper = shallow(<ConnectedApp {...defaultProps} />);
+
+    const image = wrapper.find('img');
+    expect(image.prop('src')).to.equal('logoURL');
+
+    const text = wrapper.text();
+    expect(text).to.include('Random title');
```

Comment:
minor nit: if this was changed to `...to.include(defaultProps.attributes.title)` it would make it crystal clear that the component should be showing that part of the pros in the UI.